controllers/healthcheck_controller.go

```diff
@@ -139,8 +139,8 @@ func (r *HealthCheckReconciler) processHealthCheck(ctx context.Context, req ctrl
 	if hcSpec.Workflow.Resource != nil {
 		wfNamePrefix := hcSpec.Workflow.GenerateName
 		wfNamespace := hcSpec.Workflow.Resource.Namespace
-		level := hcSpec.Level
-		sa := hcSpec.Workflow.Resource.ServiceAccount
+		//level := hcSpec.Level
```

Comment:
??

Reply:
will remove them level and sa from processHealthCheck they are used when creating rbac permissions not in this funciton